vtm.js

```diff
@@ -0,0 +1,339 @@
+                    {id: "drive", display: "Drive",value: 0},
+                    {id: "firearms", display: "Firearms", value: 0},
+                    {id: "larceny", display: "Larceny", value: 0},
+                    {id: "melee", display: "Melee", value: 0},
+                    {id: "stealth", display: "Stealth",value: 0},
+                    {id: "survival", display: "Survival",value: 0}
+                ],
+             },
+             {
+                display: "Social",
+                list: [
+                    {id: "animal ken", display: "Animal Ken",value: 0},
+                    {id: "etiquette", display: "Etiquette",value: 0},
+                    {id: "insight", display: "Insight",value: 0},
+                    {id: "intimidation", display: "Intimidation",value: 0},
+                    {id: "leadership", display: "Leadership", value: 0},
+                    {id: "performance", display: "Performance", value: 0},
+                    {id: "persuasion", display: "Persuasion", value: 0},
+                    {id: "streetwise", display: "Streetwise",value: 0},
+                    {id: "subterfuge", display: "Subterfuge",value: 0}
+                ],
+             },
+             {
+                display: "Mental",
+                list: [
+                    {id: "academics", display: "Academics",value: 0},
+                    {id: "awareness", display: "Awareness",value: 0},
+                    {id: "finance", display: "Finance",value: 0},
+                    {id: "investigation", display: "Investigation",value: 0},
+                    {id: "medicine", display: "Medicine", value: 0},
+                    {id: "occult", display: "Occult", value: 0},
+                    {id: "politics", display: "Politics", value: 0},
+                    {id: "science", display: "Science",value: 0},
+                    {id: "technology", display: "Technology",value: 0}
+                ],
+             
+             },],
+             resource: [12,7,5,3,0,0],
+             
+        }
+  }}
+})
+//SHEET
+app.component('sheet',{
+  props: ['attr', 'skills'],
+  template: `<stat-section 
+               :stats="attr"               
+               :parent="attr.parent"></stat-section>
+               <stat-section 
+               :stats="skills"               
+               :parent="skills.parent"></stat-section>
+            `
+})
+// <div>{{attr}}</div>
+
+//STAT SECTION
+app.component('stat-section',{
+  props: ['stats', 'parent'],
+  data() {
+    return { 
+      resourceCount : null,
+  }},
+  template: `<div>{{resourceCount}}</div>
+             <div class="statSection">
+               <h2>{{stats.display}}</h2>
+               <stat-category v-for="list in stats.data"
+               :categ="list"
+               :resource="this.resourceCount"
+               :scale="stats.resource.length - 1"
+               :parent="parent"></stat-category></div>`,
+  /*computed:{
```

Comment:
komentar tu neni treba, ale mozno si ho tam nechat kvoli prezentacii peterovi

Reply:
neviem ktore riesenie je lepsie, zatial nechavam obidve lebo sa mi to nechce potom prepisovat